### DISCUSSION TITLE
쓰는게 엑셀임 한셀임

### DISCUSSION THREAD

**Discussion by RimWorldKorea** [RimWorldKorea/RMK#258] [일반]
@zilrty
엑셀이면 그냥 엑셀 창에다가 csv 파일 드래그드랍하면 됨

**Comment by zilrty** [comment]
왈도서식 아니어도 상관 없음?

**Reply by RimWorldKorea** [reply]
lookup같은거 써서 매칭하면 되니까
rmk툴로 뽑히는 파일이 왈도 파일에 병합시키기 최적화된 형태임

**Reply by RimWorldKorea** [reply]
지금 위키 계속 쓰고 있으니깐 기다렸다 그거 참고해도 되고
https://github.com/RimWorldKorea/RMK/wiki

**Comment by zilrty** [comment]
엑셀 함수 섬이랑 데이트 밖에 몰라서 ㅋㅋ

대충 알려주면 알아서 해보겠음

**Reply by RimWorldKorea** [reply]
[여기](https://github.com/RimWorldKorea/RMK/blob/901e04fd34672ed49c4dca117aaf1191d0908998/Data/Dragon's%20Descent%20-%202026992161/2026992161.xlsx)서 우측 상단에 점점점 메뉴 누르면 다운로드 버튼 뜸

`Update` 시트에 csv 파일을 붙여넣고 `Main` 시트로 병합시키는 과정을 살펴볼 수 있음
내용보단 함수를 어떤 식으로 쓰는지 중점으로 보면 됨

주로 쓰는 함수는 `IF`, `ISERROR`, `IFERROR`, `VLOOKUP`, `MATCH`, `TEXTJOIN`, `TEXTSPLIT` 정도 알아두면 충분할 듯?
